saleor/plugins/base_plugin.py

```diff
@@ -379,6 +380,29 @@ def order_confirmed(self, order: "Order", previous_value: Any):
         """
         return NotImplemented
 
+    def sale_created(self, sale: "Sale", current_catalogue, previous_value: Any):
```

Comment:
Are we able to add typing to these methods? For someone who wants to use it in own plugin, this woud be very helpfull

Reply:
Yes, ofc, I was thinking even about named type like `CatalogueInfo = Dict[str,  Set[int]]` and `NodeCatalogueInfo = Dict[str, Set[str]]`. What do you think?

Reply:
Typing added.